docs/extend/extend-apiml/api-mediation-security.md

```diff
@@ -337,18 +352,7 @@ https://github.com/zowe/api-layer/blob/master/keystore/README.md#trust-certifica
 
 #### Log in to Discovery Service on localhost
 
-To access Discovery Service on localhost provide a valid client certificate.
-
-The certificate is stored in the `keystore/localhost/localhost.keystore.p12` keystore.
-
-Some utilities including HTTPie require the certificate to be in PEM format. You can find it in `keystore/localhost/localhost.pem`.
-
-Since the Discovery Service is using HTTPS, your client also requires verification of the validity of its certificate. Verification is performed by trusting the local CA certificate which is store at `keystore/local_ca/localca.cer`.
-
-The following is an example of how to access Discovery Service from CLI with full certificate validation:
-
-`http --cert=keystore/localhost/localhost.pem --verify=keystore/local_ca/localca.cer -j GET https://localhost:10011/eureka/apps/`
-
+To access Discovery Service on localhost provide valid mainframe credentials or token.
```

Comment:
this is ambiguous, maybe it would be more effective to plug in the security matrix table?

Reply:
Good idea, the table could be more clear